### ISSUE TITLE
Could not find version: latest of iTMSTransporter to download

### ISSUE THREAD

**Issue by joquerod** [fastlane/fastlane#14180]
### Question Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Question Subject
Summary:
- Currently using Fastlane 2.115.0
- I'm running the lane upload_to_testflight
- I get a successful response: "Successfully uploaded the new binary to App Store Connect"
- The App never made it to Testflight
- The console shows an error: Could not find version: latest of iTMSTransporter to download.

I'm not submitting this issue as a bug, because we are behind a proxy.
We had issues before with Fastlane trying to reach the Itunes Connect APIs, and I got help from the IT team to configure some proxies, create a whitelist of APIs to ues,  and everything started to work.
We stop using this environment for a couple of months, and now that I'm back I see this new issue.

Question: Is there an API that fastlane is trying to reach to download the latest iTMSTransporter? If that's the case we could include it in the whitelist for the current proxies.
I see in the logs the iTMSTransporter creates this directory: ~/.itmstransporter
Is that where the latest iTMSTransporter is stored?

#### Question Description
This is the lane we use:

upload_to_testflight
[testflight_log.txt](https://github.com/fastlane/fastlane/files/2824688/testflight_log.txt)

(
          username: "myusername",
          ipa: ipa_path,
          team_name: group_name,
          skip_waiting_for_build_processing: true
      )

Log attached.

**Comment by janpio** [comment]
That is probably output from iTMSTransporter, that is trying to update itself. iTMSTransporter is part of Xcode, but seems it decided it has to update itself. Because of your restrictive network configuration it can't do that and failing. 

It also seems as if the code doesn't handle that case very well, giving you the wrong success message :/

**Comment by sshaw** [comment]
@joquerod the list of hosts The Transporter connects to can be found here: https://help.apple.com/itc/transporteruserguide/#/apdATD1E112-D1E1A1303-D1E112A1126

**Comment by janpio** [comment]
(We could probably add a documentation page "fastlane in enterprise environments" or similar, that collects information like that about IPs to unblock etc.)

**Comment by joquerod** [comment]
Yes! Fastlane for enterprises needs its own section. 
@sshaw That's what I needed.
Thank you guys.

**Comment by janpio** [comment]
Could you create a new issue for that @joquerod and already include the information you already have and gave to your IT team? That would be a nice start.

**Comment by joquerod** [comment]
This issue has been resolved removing the ~/.itmstransporter folder from the machine.
We didn't have to add all the new APIs to the whitelist. We didn't try it because it will take more time working it out with the IT team.

We still get an error and I think is related to our proxy "Could not find version: latest of iTMSTransporter to download"
If we ignore the error everything seems to work fine.

Every time we ran this Jenkins job we got a successful response: "Successfully uploaded the new binary to App Store Connect" but we had to check on Itunes Connect to be sure the App was there.
The only difference after removing the cash folder is that the App is actually there!

@janpio do you still want me to create another issue? as a bug? For sure the expectation is to avoid false positives, and receive the right message even if you are behind a proxy.

**Comment by janpio** [comment]
Yes, feature or bugs report - it will be more a placeholder for "add some enterprise documentation and common gotchas" ;)
